db_release_scripts/mosip_keymgr/sql/1.2.0-release.sql

```diff
@@ -9,6 +9,11 @@ ALTER TABLE keymgr.key_alias ADD CONSTRAINT uni_ident_const UNIQUE (uni_ident);
 ALTER TABLE keymgr.key_policy_def ADD COLUMN pre_expire_days smallint;
 ALTER TABLE keymgr.key_policy_def ADD COLUMN access_allowed character varying(1024);
 
+insert into keymgr.key_policy_def(app_id, key_validity_duration, is_active, cr_by, cr_dtimes, pre_expire_days, access_allowed) values('ADMIN_SERVICES',1095,TRUE,'mosipadmin',now(),60,'NA');
+insert into keymgr.key_policy_def(app_id, key_validity_duration, is_active, cr_by, cr_dtimes, pre_expire_days, access_allowed) values('RESIDENT',1095,TRUE,'mosipadmin',now(),60,'NA');
+insert into keymgr.key_policy_def(app_id, key_validity_duration, is_active, cr_by, cr_dtimes, pre_expire_days, access_allowed) values('DIGITAL_CARD',1095,TRUE,'mosipadmin',now(),60,'NA');
```

Comment:
As part of https://github.com/mosip/keymanager/blob/1.1.5.5/db_scripts/mosip_keymgr/dml/keymgr-key_policy_def.csv I see DIGITAL_CARD already, do we need to insert this ?

Reply:
Its added in 1.1.5.5 on july 1st, seems to be a recent change there. 'DIGITAL_CARD' is added on the released branch.